spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/SparkTestBaseWithCatalog.java

```diff
@@ -80,6 +80,7 @@ public SparkTestBaseWithCatalog(
     spark.conf().set("spark.sql.catalog." + catalogName, implementation);
     config.forEach(
         (key, value) -> spark.conf().set("spark.sql.catalog." + catalogName + "." + key, value));
+    spark.conf().set("spark.sql.defaultCatalog", catalogName);
```

Comment:
Why set this for all tests? If you need it set in the new one, isn't it better to set it only in that test?

Reply:
I removed it. Just set it as a convenience when seeing what could be done with the session catalog or not.